kubernetes/resource_kubernetes_persistent_volume.go

```diff
@@ -245,6 +246,23 @@ func resourceKubernetesPersistentVolumeDelete(d *schema.ResourceData, meta inter
 		return err
 	}
 
+	err = resource.Retry(d.Timeout(schema.TimeoutDelete), func() *resource.RetryError {
```

Comment:
Since this is retying a `Get` action, should this maybe use `schema.TimeoutRead` instead?

Reply:
Hmm, I've replicated what's already done everywhere else.

Reply:
Apparently, this timeout matches the name of the enclosing function.

Reply:
I believe you don't have any other choice than doing a read (Get or List) to find out if a deletion or update happened.

Reply:
Makes sense then.